protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -3481,3 +3506,151 @@ message LiquidationPrice {
   // Liquidation price assuming sell orders start getting filled.
   string including_sell_orders = 3;
 }
+
+// Request to get the current referral program
+message GetCurrentReferralProgramRequest {}
+
+// Response containing the current referral program
+message GetCurrentReferralProgramResponse {
+  // Referral program currently activated on the network.
+  ReferralProgram current_referral_program = 1;
+}
+
+// Referral program for reporting purposes differs slightly from the referral program
+// message in the ReferralProgramCreated and ReferralProgramUpdated events, as it includes
+// an optional timestamp that is set when the referral program has ended.
+message ReferralProgram {
+  // Incremental version of the program. It is incremented after each program
+  // update.
+  uint64 version = 1;
+  // Unique ID generated from the proposal that created this program.
+  string id = 2;
+  // Defined tiers in increasing order. First element will give Tier 1, second
+  // element will give Tier 2, and so on.
+  repeated vega.BenefitTier benefit_tiers = 3;
+  // Timestamp as Unix time in seconds, after which when the current epoch ends, the
```

Comment:
This is described as nanoseconds for GQL. Are they actually different?

Reply:
The time provided is as Unix time in seconds from core. The Time format from GQL provides time to the nanosecond precision, even though that precision is not provided in the original data, so it will just display a lot of 0's for the nanosecond timestamp. All time in GQL is formatted as RFC3339Nano.